service_user.go

```diff
@@ -26,6 +26,7 @@ type (
 	}
 
 	AccessControl struct {
+		M3Group                  *string  `json:"m3_group"`
```

Comment:
Probably there is an 'omitempty' missing here, so we dont get `"m3_group": null"` in the request!

Reply:
I can add the omitempty, but it will lead to the incapacity to remove the group from an existing user since there will be no explicit null

Reply:
I looked it up in the API code its functionally identical, so feel free to dismiss my comment!

Reply:
Yeah PR is approved, i would merge it with the omitempty to be consistent, if you are ok with it.